trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/extract-query-and-session-data.test.ts

```diff
@@ -7,59 +7,61 @@ jest.mock('./validate-facility-id-query');
 
 describe('extractQueryAndSessionData', () => {
   const ORIGINAL_URL = '/original-url';
-  const FACILITY_ID_QUERY = '1234';
+  const PREMIUM_PAYMENTS_FACILITY_ID_QUERY = '1234';
 
   beforeEach(() => {
     jest.resetAllMocks();
 
     jest.mocked(handleRedirectSessionData).mockReturnValue({
-      tableDataError: undefined,
+      premiumPaymentsTableDataError: undefined,
```

Comment:
Instead of setting an undefined value, can we declare `premiumPaymentsTableDataError` as an optional type?

Reply:
Made this change changes, removed this from the stub as it's no longer required (as its now optional)